yang-module-versioning/draft-ietf-netmod-yang-module-versioning.xml

```diff
@@ -683,19 +672,19 @@ module ietf-yang-library-revisions {
   // RFC Ed.: please replace label version with 1.0.0 and
   // remove this note.
 
-  revision 2021-11-04 {
-    rev:label "1.0.0-draft-ietf-netmod-yang-module-versioning-05";
+  revision 2024-02-10 {
+    rev:label "1.0.0-draft-ietf-netmod-yang-module-versioning-11";
     description
       "Initial revision";
     reference
       "XXXX: Updated YANG Module Revision Handling";
   }
   
-  augment "/yanglib:yang-library/yanglib:schema" {
+  container conformance {
+    config false;
     description
-      "Augmentations to the ietf-yang-library module to indicate how
-       deprecated and obsoleted nodes are handled for each datastore
-       schema supported by the server.";
+      "To indicate how deprecated and obsoleted nodes are handled by
+      the server.";
```

Comment:
As agreed in Feb 13th meeting, revert to augment. But remove the "for each datastore schema" part.